### ISSUE TITLE
CLI should additionally detect `prisma/prisma.yml`

### ISSUE THREAD

**Issue by schickling** [prisma/prisma#3250]
**Is your feature request related to a problem? Please describe.**
It's a very common use case that you're putting your `prisma.yml` + `datamodel.prisma` in a `prisma` subfolder. Every time you're now trying to use the Prisma CLI (e.g. run `prisma generate`) you need to go into the subdirectory via `cd prisma` first.

Another common workaround is to add the following to your `package.json` when using Node which allows the `yarn prisma` alias:

```json
  "scripts": {
    "prisma": "cd prisma && prisma"
  }
```

However this workaround leads to a handful of other pitfalls and should be avoided IMO.

**Describe the solution you'd like**
The Prisma CLI should automatically detect whether there's a `prisma` folder in the current directory that contains a `prisma.yml` file. If there's a `prisma.yml` file in the current directory, this file take precedence.

**Additional context**
In the future we should consider introducing a `--file / -f <prisma-yml-file-path>` argument as well (similar to [docker-compose](https://docs.docker.com/compose/reference/overview/#command-options-overview-and-help)). However, the `-f` argument name currently would clash with the `prisma deploy -f` command. (See https://github.com/prisma/prisma/issues/3140)

**Comment by danielmahon** [comment]
@schickling Im not able to get `yarn prisma deploy` to do anything using the default `node-graphql` example. After following the instructions it just returns this:
```
graphql(master) > yarn prisma deploy
yarn run v1.9.4
$ cd prisma && prisma deploy
✨  Done in 0.07s.
```

If I manually `cd prisma` and then `prisma deploy` it works fine. Any idea what im doing wrong?

**Comment by schickling** [comment]
I'd advice to not use the `yarn prisma` approach and wait until this feature lands. It should be implemented and released shortly! 👍

**Comment by jasonkuhrt** [comment]
@schickling Shouldn't this issue be closed now?

**Comment by marktani** [comment]
@jasonkuhrt the feature has been added to `beta`, this issue will be closed once `beta` is merged into `master` (next Tuesday).

**Comment by schickling** [comment]
This feature is now available in the latest versions.

**Comment by vdiaz1130** [comment]
What is the flag to specify the file path for the prisma.yml file when running prisma deploy? I tried --file which failed.
